docs/source/start.rst

```diff
@@ -61,7 +61,8 @@ and Quil Compiler from the bare-bones package, you will have to install the prer
 
 Installing on macOS
 -------------------
-Mount the file ``forest-sdk.dmg`` by double clicking on it in your email. From there, open ``forest-sdk.pkg`` by
+Download the macOS distribution by clicking on the appropriate link on the `SDK download page<https://qcs.rigetti.com/sdk-downloads>`__.
```

Comment:
Are these supposed to be single underscores? Sphinx is failing with some obscure error.

Reply:
not sure about the double-underscore, but you definitely need a space before the `<`? ditto just below.